content/docs/command-reference/import.md

```diff
@@ -137,6 +137,38 @@ Several of the values above are pulled from the original stage file
 subfields under `repo` are used to save the origin and version of the
 dependency, respectively.
 
+### Importing from non-DVC repositories
```

Comment:
Please note that we use the word "Example" for example headers. I would also use the term "Git" here for explicitness
```suggestion
### Example: Importing from any Git repository
```

Reply:
- [ ] And please move the whole example to the bottom. The existing 2 seem more important to me. (Unless you have a different impression, we can discuss.)